db/routines.sql

```diff
@@ -0,0 +1,50 @@
+CREATE OR REPLACE FUNCTION lock_preset(IN preset_name CHARACTER)
```

Comment:
I think `CHARACTER` is fixed length according to Postgres docs: https://www.postgresql.org/docs/9.0/datatype-character.html
IMO it would be better to use VARCHAR here

Reply:
You have absolutely right, fixed.